### PR TITLE
Coc

### PR DESCRIPTION
**Pull request by mdsams**

### CODE REVIEW

**Comment by cosmo3769** [comment]
@mdsams Hey, it looks good. Some changes to be done: 

* The content is not responsive in other screen(From Anti Harassment Policy, it is slightly right shifted.)
* The image is not responsive too. It is taking full width of the screen when inspecting. 
* The image box shadow is perfect in desktop, but when inspecting the box shadow is overlapping with faq section.

**Review by cosmo3769 — CHANGES_REQUESTED** [review]
@mdsams Hey, it looks good. Some changes to be done:

The content is not responsive in other screen(From Anti Harassment Policy, it is slightly right shifted.)
The image is not responsive too. It is taking full width of the screen when inspecting.
The image box shadow is perfect in desktop, but when inspecting the box shadow is overlapping with faq section.

File: src/Components/Coc/Coc.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+const Coc = () => {
+  return (
+    <>
+      <Title heading="Code Of Conduct" />
+      <div className="grid sm:grid-cols-1 md:grid-cols-2 place-items-center bg-opacity-50">
+        <div className="w-full lg:w-3/4 my-0 mx-auto pl-20">
+          <h3 className="text-3xl py-5 font-bold">AntiHarassment Policy</h3>
+          <h3 className="text-regalBlue text-s font-bold">
+            Why do we have an official Anti-Harassment policy for Developer Student
```
Comment (cosmo3769) [inline src/Components/Coc/Coc.tsx]: Have to change from Developer Student Clubs NSEC to Google Developer Group.

**Review by cosmo3769 — CHANGES_REQUESTED** [review]

**Review by cosmo3769 — APPROVED** [review]
LGTM.

**Review by avinashkranjan — APPROVED** [review]
LGTM..

**Pull request merged**